test/interface.jl

```diff
@@ -282,6 +282,94 @@ struct NonInvertibleBijector{AD} <: ADBijector{AD} end
         @test f_t == f_a
     end
 
+    @testset "Stacked <: Bijector" begin
```

Comment:
Can we add a simple test that do something like below?
```julia
x = ones(5)
sb = vcat(Exp(), Log(), ...) # stack some simple bijectors
@test sb(x) == [...] # some simple exact answers
```
```

Reply:
Is this done?

Reply:
Yup: L339